scapy/layers/bluetooth4LE.py

```diff
@@ -342,14 +342,16 @@ class BTLE_CONNECT_REQ(Packet):
 
 
 BTLE_Corp_IDs = {
+    0x9: 'Infineon Technologies AG',
     0xf: 'Broadcom Corporation',
     0x59: 'Nordic Semiconductor ASA'
+    0x131: 'Cypress Semiconductor'
 }
 
 
 BTLE_BTLE_CTRL_opcode = {
-    0x00: 'LL_CONNECTION_UPDATE_REQ',
-    0x01: 'LL_CHANNEL_MAP_REQ',
+    0x00: 'LL_CONNECTION_UPDATE_IND',
```

Comment:
Sorry for the delay in reviewing these changes. Could you explain why you are modifying the names for values 0x00 and 0x01?

Reply:
because Bluetooth SIG updated those names in latest Spec(https://www.bluetooth.com/specifications/specs/core-specification/)

Reply:
Understood. I think that we should keep the existing names for the 0x00 and 0x01 values to make sure that no code relying on theses values will break.